pkg/lib/dynatrace/dynatrace.go

```diff
@@ -782,6 +826,31 @@ func (ph *Handler) HasDashboardChanged(keptnEvent *common.BaseKeptnEvent, dashbo
 	return true
 }
 
+/**
+ * Parses the filtersPerEntityType dashboard definition and returns the entitySelector query filter - the return value always starts with a , (comma)
+ * return example: ,entityId("ABAD-222121321321")
+ */
+func (ph *Handler) GetEntitySelectorFromEntityFilter(filtersPerEntityType map[string]map[string][]string, entityType string) string {
```

Comment:
Please add a unit test for this new function

Reply:
I added a test case for this method